runtime/src/parachains.rs

```diff
@@ -381,7 +381,7 @@ impl<T: Trait> Module<T> {
 
 			// track which voters have voted already, 1 bit per authority.
 			let mut track_voters = bitvec![0; authorities.len()];
-			for (auth_id, validity_attestation) in &candidate.validity_votes {
+			for (auth_index, validity_attestation) in &candidate.validity_votes {
 				// protect against double-votes.
 				match validator_group.iter().find(|&(idx, _)| &authorities[*idx] == auth_id) {
```

Comment:
I think it would be `validator_group.iter().find(|&(idx, _)| *idx == auth_index)`